Makefile

```diff
@@ -160,7 +131,7 @@ go-test:
 	./scripts/go-test.sh
 
 .PHONY: go-test-coverage
-go-test-coverage:
+go-test-coverage: embed-files
```

Comment:
Same question as before, why is this dependency required for unit testing?

Reply:
Because otherwise the build fails if it cannot import the embedded files

Reply:
I see. Does this mean embedded files are always required to be built even for unit test and static analysis or is this just a caveat in our build workflow?

Reply:
The go build command fails if the embedded file is not available. So if you do just `go test` you get a build error message 
```
./scripts/test-w-coverage.sh
cmd/cli/install.go:65:12: pattern chart.tgz: no matching files found
package github.com/openservicemesh/osm: build constraints exclude all Go files in /home/davinci/osm
go test returned non-zero
Makefile:135: recipe for target 'go-test-coverage' failed
make: *** [go-test-coverage] Error 1
```